tox.ini

```diff
@@ -15,4 +15,7 @@ deps=
     statistics
     six
 
-commands= python bench.py --n-runs 2 --upload_result {posargs}
```

Comment:
Right now ``tox`` command can be used to run any benchmarks, but after this change only specific benchmarks will be run. I agree that current default ``tox`` command is not great for running on travis due to ``--n-runs 2`` and especially ``--upload_result``, so maybe it makes sense to leave just ``python bench.py {posargs}`` here, and call all benchmarks we want to run one by one from travis?

Even better would be if by default, all benchmarks would be run with a single tox command, and individual benchmarks could be specified optionally, but this is probably out of scope for this PR.

Reply:
I made a new commit to call the tests from Travis. But I like the idea of running the tests with a single tox command. Can you elaborate more on how you want to do that? :)

Reply:
Thanks, I'll check it a bit later! 

> I like the idea of running the tests with a single tox command. Can you elaborate more on how you want to do that? :)

Not quite sure, I never used click myself. With argparse, I would probably make bench.py accept multiple positional arguments (names of the benchmarks to run), and make it default to all benchmarks. Alternatively, default value can live in tox.ini, but still bench.py must accept multiple benchmark names in this case.

Reply:
With my new commit, I think I made it :) I tested it and the bench.py now can take multiple commands and we can call tox once.